pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -219,11 +223,17 @@ func (rc *reconciler) reconcile() {
 				continue
 			}
 
-			// Trigger detach volume which requires verifing safe to detach step
+			// insert a detach operation start time into cache if needed for the volume for metrics reporting
+			// note that the plugin name has been set to "" as there is no visibility at this moment to which
+			// plugin/csi driver would be conducting the operation. It will be populated later when detach
+			// operation function is created and plugin name then could be decided
+			rc.operationStartTimeCache.AddIfNotExist(operationexecutor.CreateAttachDetachOperationUniqueKey(attachedVolume.VolumeName, attachedVolume.NodeName), "", "volume_detach")
```

Comment:
do we need to start the time now, or can we wait until inside operation generator when we actually have the plugin name?